lib/pulfalight/digital_object.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # Add a role attribute to this class
-class Arclight::DigitalObject
+class Pulfalight::DigitalObject < Arclight::DigitalObject
```

Comment:
Why move this to lib, out of curiosity? Now we have models in two places. Does this work in like, `app/models/pulfalight/digital_object.rb`?

Reply:
I was following the pattern in https://github.com/projectblacklight/arclight/blob/master/lib/arclight/digital_object.rb